baipw/middleware.py

```diff
@@ -4,9 +4,11 @@
 from django.core.exceptions import PermissionDenied
 from django.utils.module_loading import import_string
 
+from ipware import get_client_ip
```

Comment:
**Suggestion:** I think it'll be nicer if we can see where this is coming from in the code below - we could use `ipware.get_client_ip` instead. I believe it makes the code clearer. What do you think?
```suggestion
import ipware
```